lib/hal/fileio.c

```diff
@@ -75,7 +75,8 @@ static char *getCurrentDirString()
 
 /**
  * Converts a DOS-style path (eg. "c:/foo/bar.txt") to a XBOX-style
- * path (eg. "\Device\Harddisk0\Partition2\foo\bar.txt")
+ * path (eg. "\Device\Harddisk0\Partition2\foo\bar.txt").
+ * Returns early if the path is alread in XBOX-style.
```

Comment:
nit: This function doesn't just return early, it will simply use the input string as-is when copying to xboxFilename.

Reply:
Fixed, thanks!